agent/connect/ca/provider_test.go

```diff
@@ -51,7 +52,7 @@ func TestStructs_CAConfiguration_MsgpackEncodeDecode(t *testing.T) {
 				Config: map[string]interface{}{
 					"PrivateKey":          "key",
 					"RootCert":            "cert",
-					"RotationPeriod":      "5m",
+					"RotationPeriod":      "5m", // old unused field
```

Comment:
I don't see the point of keeping this in the test, but I'm probably missing something 🙂

Reply:
It's not strictly necessary, but since this test is for `MsgpackEncodeDecode`, I thought it would be useful to keep it here to show that we could still properly decode values that were encoded with a previous version.